examples/island/Island.cpp

```diff
@@ -240,11 +240,10 @@ sf::Vector2f computeNormal(float left, float right, float bottom, float top)
 /// coordinates.
 ///
 ////////////////////////////////////////////////////////////
-const auto scalingFactors = sf::Vector2f(windowSize).componentWiseDiv(sf::Vector2f(resolution));
-
 sf::Vertex computeVertex(sf::Vector2u position)
 {
-    sf::Vertex vertex;
+    static const auto scalingFactors = sf::Vector2f(windowSize).componentWiseDiv(sf::Vector2f(resolution));
```

Comment:
These could all be made `constexpr`

Reply:
this is not related to the pr though and it would require changing the `windowSize` to be constexpr which I would constexpr heavily unrelated

Reply:
Yes, this is why I mentioned it as a drive by comment and I did not request changes.

Reply:
https://github.com/SFML/SFML/commit/5be64c3d6c2b9fccdf8ee5f4182f5e7cb55e69bd

I have a commit for this which might land in a future PR so this slight improvement won't be forgotten.